src/calendar-v1/CalendarGrid.ts

```diff
@@ -70,43 +73,91 @@ export const useCalendarGrid = createHook<
       focusNextWeek,
       focusPreviousWeek,
       calendarId,
+      setAnchorDate,
     } = options;
     const ref = useRef<HTMLElement>(null);
 
-    const onKeyDown = createOnKeyDown({
-      onKeyDown: htmlOnKeyDown,
-      preventDefault: true,
-      keyMap: (event: KeyboardEvent) => {
-        const shift = event.shiftKey;
```

Comment:
Any particular reasons for removing createOnKeyDown? 🤔

Reply:
Nope, I was debugging " " issue and forgot to change it back.